docs/docs/dev_docs/tutorials/token_portal/depositing_to_aztec.md

```diff
@@ -8,7 +8,20 @@ In this step, we will write our token portal contract on L1.
 
 In `l1-contracts/contracts` in your file called `TokenPortal.sol` paste this:
 
-#include_code init /l1-contracts/test/portals/TokenPortal.sol solidity
+```solidity
```

Comment:
Can you update the comments in `/l1-contracts/test/portals/TokenPortal.sol` to get the imports instead of hardcoding them? This could break in the future.

Reply:
the paths are relative so it doesn't work. our L1 contracts & absolute paths haven't changed once since I joined so I assume they will very rarely change

Reply:
ah i didn't catch that. makes sense :+1: